### ISSUE TITLE
Feature Request: How to use webpack, without laravel-elixer, or eject button etc.

### ISSUE THREAD

**Issue by TheLarkInn** [laravel/docs#2728]
Hey would love to see a guide or instructions for how to implement you're own webpack configuration. (Would this just involve not using laravel-elixir?). I'm more than happy to help setup a PR on this but didn't know what @taylorotwell and friends' opinions are on this.

Although biased opinion, I prefer to use webpack's `webpack.config.js` at the root of a project, but wasn't sure if there are specific concerns or conflicts through doing this with laravel internals for managing css or js and other assets. (I apologize I've not scratched much below the surface working on laravel).

**Comment by bobbybouwmann** [comment]
It sounds to me that this is more something for a blogpost than the Laravel documentation. You can probably find a lot other tutorials on the subject which can be used in any project. Laravel won't hold you back on that :)

Laravel supports webpack through Elixir, but doesn't go in deep on the details for custom stuff. This is because Elixir is meant for the 80% of the use cases where the basic setup is just fine. If you need something special it's easier to make it yourself I guess ;)

**Comment by TheLarkInn** [comment]
Absolutely. Infact this has spawned the creation of github.com/mutualofomaha/multipage-webpack-plugin

We are using laravel but no SPA features rather server routes pages, so to have awesome support we have to go through some creative steps to create html-webpack-plugin driven twig/blade partials that include the scripts we need only for that entry point. If the author(s) of laravel-elixer-webpack-official would like to contribute some insight or learnings from the current usage or architectures for webpack multipage laravel apps, I would be more than happy to have their input // cc @taylorotwrll

**Comment by TheLarkInn** [comment]
CC @JeffreyWay @taylorotwell
